Enterprise-CMCS/macpro-mdct-mfp
# ui updates to sar dashboard table

### Description
This PR is to update the SAR dashboard table to include Target Populations and update general styling. The due date, last edited, etc data is not expected to be correct in this PR.
![Screenshot 2023-10-17 at 10 23 00 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/22454493/cb516725-496d-4c59-b1b1-e04022389484)


### Related ticket(s)


CMDCT-3017

---
### How to test
Login and see that all expected data is there in the SAR table in regular view and mobile view. Also see that the table for the work plan has not been modified.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
